src/Std/Data/DHashMap/Internal/WF.lean

```diff
@@ -486,12 +486,26 @@ theorem wfImp_insert [BEq α] [Hashable α] [EquivBEq α] [LawfulHashable α] {m
 
 /-! # `alter` -/
 
+theorem mem_of_perm {l l' : List α} {x : α} (h : l ~ l') : x ∈ l ↔ x ∈ l' := by
```

Comment:
https://loogle.lean-lang.org/?q=List.Perm%2C+Membership.mem+ tells me that this exists as `List.Perm.mem_iff`.

Reply:
Thanks! I was already wondering. Just FYI before you lose too much time: I'm still far from review-ready...